cmd/blocking-issue-creator/main.go

```diff
@@ -44,6 +45,7 @@ func gatherOptions() options {
 	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 	fs.StringVar(&o.username, "username", "", "Username to use when communicating with GitHub.")
 	fs.StringVar(&o.tokenPath, "token-path", "", "Path to token to use when communicating with GitHub.")
+	fs.StringVar(&o.ghGraphqlEndpoint, "github-graphql-endpoint", "https://api.github.com/graphql", "GH GraphQL endpoint URL")
```

Comment:
Those are not cacheable at all, this will only help with getting metrics. Ratelimiting might help to not hit the abuse limit, but if possible it might be better to replace the graphql requests with v3 api requests so we can cache

Reply:
Avoiding hitting the abuse limit is my primary goal and being counted in metrics is a good thing too. Given how trivial this PR is compared to making it use a differnt client, can we try merging this?

Reply:
yeah but this change doesn't do anything that would help with not hitting the abuse limit

Reply:
I lived under impression that the proxy itself does some throttling to avoid abuse limits :facepalm: